crates/bevy_ecs/src/system/function_system.rs

```diff
@@ -428,6 +430,8 @@ where
         self.param_state = Some(Param::init_state(world, &mut self.system_meta));
     }
 
+    /// For all current [`Archetype`]s present in the [`World`],
+    /// registers the archetype-components accessed by this [`SystemParam`] (if applicable).
```

Comment:
```suggestion
```

This is a trait method, its documentation is on the trait definition.

Reply:
i see, ty